collectors/python.d.plugin/python.d.conf

```diff
@@ -88,6 +88,7 @@ nginx_log: no
 # redis: yes
 # rethinkdbs: yes
 # retroshare: yes
+# riak: yes
```

Comment:
i think we need to name it `riakkv` or `riakkv`

Reply:
Renamed in d7f04df1db2cb0967b8482fa0a4ecf6a50eb1795.